src/Polysemy/Scoped.hs

```diff
@@ -36,26 +36,26 @@ import Polysemy.Internal.Tactics
 -- use the 'Tactical' environment and transforms the effect into other effects
 -- on the stack.
 interpretScopedH ::
-  ∀ param resource effect r .
+  ∀ resource param effect r .
```

Comment:
Does this param order change make sense any longer now `resource` is no longer part of the effect type? Or perhaps I'm getting confused and that's exactly why you made the change? If the latter, please explain your reasoning.

Reply:
my only reason was that most of the others had this order.
I guess the resource must always be fixed in an interpreter, while the param may be polymorphic. should we reorder them all to have `param` as the first one? and `resource` as the second or `effect`?

Reply:
I feel the order doesn't have as much meaning now. As the one most experienced with `Scoped`, it's up to you what the order should be.

Reply:
I did a grep and I'm not using type applications anywhere in my projects for those interpreters…I think we can leave it like it is.